impls/src/serialization.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 The Epic Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Responsible for handling the serialization and deserialization of structs common to the database
+
+use epic_wallet_libwallet::{
+	AcctPathMapping, Context, OutputData, ScannedBlockInfo, TxLogEntry, WalletInitStatus,
+};
+use serde::Serialize;
+use serde_json::Result;
+
+/// Stores all the structs with data that needs to be stored on the database
+/// This enum implements traits used on serde_json
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(untagged)]
+pub enum Serializable {
+	TxLogEntry(TxLogEntry),
+	AcctPathMapping(AcctPathMapping),
+	OutputData(OutputData),
+	ScannedBlockInfo(ScannedBlockInfo),
+	WalletInitStatus(WalletInitStatus),
```

Comment:
`ScannedBlockInfo` and `WalletInitStatus` aren't being used, right? We can remove these

Reply:
They are:
- ScannedBlockInfo: https://github.com/EpicCash/epic-wallet/blob/5f94f1ef02e1772bbaab889b465e41552838fd01/impls/src/backends/lmdb.rs#L495-L519
- WalletInitStatus: https://github.com/EpicCash/epic-wallet/blob/5f94f1ef02e1772bbaab889b465e41552838fd01/impls/src/backends/lmdb.rs#L521-L536

I have never seen a ScannedBlockInfo, but WalletInitStatus is one of the first structs created on the database